app/models/genre.rb

```diff
@@ -0,0 +1,15 @@
+class Genre < ApplicationRecord
+  has_and_belongs_to_many :books
+
+  validates :name,
+            presence: true,
+            uniqueness: true
+
+  scope :ordered, -> { order('LOWER(genres.name)') }
+
+  scope :search, -> (search_term) do
+    search_term_with_wildcards = "%#{search_term}%"
+    where("name LIKE ?",
+    search_term_with_wildcards)
```

Comment:
I think this might reader nicer on one line - I really only try to split things across multiple lines when the line gets long (80+ characters)

```ruby
where("name LIKE ?", search_term_with_wildcards)
```

if I split a long line, I'll try to indent the next lines to make it clear that they belong to the above lines:

```ruby
my_cool_method(
  arg_1,
  arg_2,
  arg_3,
)

# or

my_cool_assignment =
  my_cool_method(
    line_1,
    line_2,
    line_3,
  )
```